va_explorer/templates/partials/_va_table.html

```diff
@@ -3,6 +3,7 @@
 <table class="table table-sm">
   <thead>
     <tr>
+      <th><a href="?{% sort_url 'Id10010' direction='-' %}">ID</a></th>
```

Comment:
```suggestion
      <th><a href="?{% sort_url 'id' direction='-' %}">ID</a></th>
```
I think this is the actual desired behavior here

Reply:
fixed, I copy-pasted from a line below and forgot to edit